test/server/cards/05_LOF/leaders/AvarKrissMarshalOfStarlight.spec.ts

```diff
@@ -0,0 +1,236 @@
+            });
+        });
+
+        describe('Avar Kriss\'s deployed constant ability', function () {
+            it('should give her +0/+4 and Overwhelm when the player has the Force', async function () {
+                await contextRef.setupTestAsync({
+                    phase: 'action',
+                    player1: {
+                        leader: { card: 'avar-kriss#marshal-of-starlight', deployed: true },
+                        hasForceToken: true
+                    },
+                    player2: {
+                        groundArena: ['battlefield-marine']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                expect(context.avarKriss.getPower()).toBe(8);
+                expect(context.avarKriss.getHp()).toBe(10);
+
+                context.player1.clickCard(context.avarKriss);
+                context.player1.clickCard(context.battlefieldMarine);
+
+                expect(context.p2Base.damage).toBe(5);
+            });
+
+            it('should do nothing when the player does not have the Force', async function () {
+                await contextRef.setupTestAsync({
+                    phase: 'action',
+                    player1: {
+                        leader: { card: 'avar-kriss#marshal-of-starlight', deployed: true }
+                    },
+                    player2: {
+                        groundArena: ['battlefield-marine']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                expect(context.avarKriss.getPower()).toBe(4);
+                expect(context.avarKriss.getHp()).toBe(10);
+
+                context.player1.clickCard(context.avarKriss);
+                context.player1.clickCard(context.battlefieldMarine);
+
+                expect(context.p2Base.damage).toBe(0);
+            });
+
+            it('should work when the Force is gained and lost', async function () {
+                await contextRef.setupTestAsync({
+                    phase: 'action',
+                    player1: {
+                        leader: { card: 'avar-kriss#marshal-of-starlight', deployed: true },
+                        base: 'mystic-monastery',
+                        hand: ['cure-wounds']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                expect(context.avarKriss.getPower()).toBe(4);
+                expect(context.avarKriss.getHp()).toBe(10);
+
+                context.player1.clickCard(context.mysticMonastery);
+                expect(context.avarKriss.getPower()).toBe(8);
+                expect(context.avarKriss.getHp()).toBe(10);
+
+                context.player2.passAction();
+                context.player1.clickCard(context.cureWounds);
+
+                expect(context.avarKriss.getPower()).toBe(4);
+                expect(context.avarKriss.getHp()).toBe(10);
+            });
+        });
```

Comment:
can you add a test without force token but 28 hp base, player should gains token before damage step, so she should have the buff